data/adapters/encode_enhancer_gene_adapter.py

```diff
@@ -105,9 +105,17 @@ def process_file(self):
                         'chr': chr,
                         'start': start,
                         'end': end,
-                        'class': class_name,
+                        'type': 'candidate_cis_regulatory_element',
                         'source': self.source,
                         'source_url': self.source_url
                     }
 
+                    if class_name != 'enhancer':
```

Comment:
should be `if class_name == 'enhancer':`

Reply:
oh good catch!